### PR TITLE
Support kube config

### PR DESCRIPTION
**Pull request by yuanying**

### CODE REVIEW

**Comment by yuanying** [comment]
Fixes #3

**Comment by yuanying** [comment]
@mbohlool Could you check this?

**Comment by yuanying** [comment]
@abstrctn Thanks for your reviewing!

> It would be great to add support for the user's auth-provider config attribute in a later update to support the credentials added by default when connecting to GKE clusters (I'd be happy to make a PR).

I'm happy to hear that! Supporting auth-provider is necessary I think.

File: kubernetes/lib/kubernetes/utils.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  end
+
+  #
+  # Loads authentication and cluster information from kube-config file
+  # and stores them in Kubernetes::Configuration.
+  # @param config_file [String] Path of the kube-config file.
+  # @param context [String] Set the active context. If is set to nil, current_context from config file will be used.
+  # @param client_configuration [Kubernetes::Configuration] The Kubernetes::Configuration tp set configs to.
+  def load_kube_config(
+    config_file=ENV['KUBE_CONFIG'],
```
[the remark this answers is not in the document]
Comment (yuanying) [reply kubernetes/lib/kubernetes/utils.rb]: Done!

**Comment by mbohlool** [comment]
I will be happy to merge this when @abstrctn lgtm it. From my limited ruby knowledge it looks good.

**Comment by abstrctn** [comment]
Thanks, LGTM! I'll open another issue for auth-provider support so we can frame next steps there.

**Pull request merged**